src/controllers/queryRunner.ts

```diff
@@ -222,6 +223,31 @@ export default class QueryRunner {
         this.eventEmitter.emit('batchComplete', batch);
     }
 
+    /**
+     * Refreshes the webview panel with the query results when tabs are changed
+     */
+    public async refreshQueryTab(): Promise<boolean> {
+        for (let batchId = 0; batchId < this.batchSets.length; batchId++) {
+            const batchSet = this.batchSets[batchId];
```

Comment:
I haven't heard of a concept as a "batchSet", I think it's just a batch?  Unless this variable contains a set of batches...